pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -248,30 +252,25 @@ func ingest(ctx context.Context, execCtx sql.JobExecContext, ingestionJob *jobs.
 	}
 	if reverted {
 		log.Infof(ctx, "job completed cutover on resume")
-		return nil
+		return completeIngestion(ctx, execCtx, ingestionJob)
```

Comment:
q: how come we didn't completeIngestion here before? why is this different with your change?

Reply:
> how come we didn't completeIngestion here before? 

because, as far as i could tell, we had a nasty bug before where didn't actually complete the ingestion job properly, hence this PR! In other words, whenever we revert, we must also do cleanup.